remix-analyzer/src/solidity-analyzer/modules/selfdestruct.ts

```diff
@@ -6,7 +6,7 @@ import { AnalyzerModule, ModuleAlgorithm, ModuleCategory, ReportObj, ContractHLA
 
 export default class selfdestruct implements AnalyzerModule {
   name: string = `Selfdestruct: `
-  description: string = `Beware of caller contracts`
+  description: string = `Dependent contracts can be inoperable`
```

Comment:
`Selfdestructing a contract will break all the contract that use it` ?

Reply:
doesn't it seem a bit long

Reply:
That's the description, I find it ok to have a bit more detail

Reply:
I agree with Yann - but there is a small typo there:
Selfdestructing a contract will break all the contracts that use it